frontend_csv/src/scanner/token.rs

```diff
@@ -1,5 +1,26 @@
-pub enum CSVToken {
+pub enum CSVTokenType {
     Comma,
     MsgTy,
     MsgData,
+    NewLine,
+    TlvType,
+    TlvData,
+    ShortChannelId,
+    Sha256,
+    U16,
+    U32,
+    U64,
+    ChannelId,
+    Signature,
+    Point,
+    ChainHash,
+    Byte,
+    BigSize,
+    LiteralString,
+    None,
+}
+
+pub struct CSVToken {
+    pub _type: CSVTokenType,
+    pub _value: String,
 }
```

Comment:
```suggestion
pub struct CSVToken {
    pub type: CSVTokenType,
    pub value: String,
}
```

There is the pub keyword, why here we are using the `_`?